### DISCUSSION TITLE
vue-i18n 中英文翻译

### DISCUSSION THREAD

**Discussion by DSTBP** [YunYouJun/valaxy#434] [Q&A] [answered]
请问一下，vue-i18n 怎么对 defineValaxyConfig 中的 title、defineSiteConfig 中的 title、description 以及 文章中的标题进行中英文翻译。
---
title: Hello, Valaxy!
---
操作文档有点没看懂😂

**Comment by YunYouJun** [comment, marked as answer]
`title_zh` 会读取 `locales/zh-CN.yml` 中的翻译文案。

`defineSiteConfig` 中的 `title` 忘了，晚点再看。

**Reply by DSTBP** [reply]
title_zh 会读取 locales/zh-CN.yml 中的翻译文案。这个请问具体是什么操作呢?
![image](https://github.com/user-attachments/assets/a6386f50-dc4e-4751-9308-30218d9a0144)

**Reply by YunYouJun** [reply]
```md
---
title: Your Title
title_zh: 你的标题
---

```

**Reply by DSTBP** [reply]
好的好的，非常感谢

**Reply by DSTBP** [reply]
请问一下，中英翻译似乎不对首页的标题起作用：
![image](https://github.com/user-attachments/assets/2c0a1661-4aa2-45b7-8c71-c630255b72e5)
